sdk/keyvault/azure-keyvault-administration/CHANGELOG.md

```diff
@@ -17,6 +17,7 @@
 ### Other Changes
 - Documentation improvements 
   ([#25039](https://github.com/Azure/azure-sdk-for-python/issues/25039))
+- Python 3.6 is no longer supported. Please use Python version 3.7 or later.
```

Comment:
There have been some releases since this PR was opened -- could I ask you to rebase this PR with the latest `main` branch and move all changelog notes to the newest, unreleased version?

Reply:
Sure, I've rebased the PR and moved all changelog notes to the newest unreleased version. It is also true that we can close #25902 in favor of this PR. Thank you @mccoyp , have a wonderful weekend!